prompt_toolkit/filters/cli.py

```diff
@@ -145,7 +145,7 @@ class HasArg(Filter):
     Enable when the input processor has an 'arg'.
     """
     def __call__(self, cli):
-        return cli.input_processor.arg is not None
+        return cli.input_processor._arg is not None
```

Comment:
I think this line can stay like it was, right? The "public" api of InputProcessor should not change, or at least we should not access a private attribute from here.